dataherald/tests/db/test_db.py

```diff
@@ -19,6 +19,13 @@ def __init__(self, system: System):
                 "ssh_settings": None,
             }
         ]
+        self.memory["instructions"] = [
+            {
+                "_id": "64dfa0e103f5134086f7090c",
```

Comment:
I see we are adding an instruction but are we testing the 4 new instruction endpoints?

Reply:
@aazo11 we basically don't test the instructions, all of the pytests that we have are only empty classes, almost for all of the classes